apps/site/assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -248,16 +273,25 @@ const ScheduleDirection = ({
           </a>
         </>
       )}
-      {!isSubwayRoute(route)
-        ? lineState.data &&
-          lineState.data[0] && (
-            <LineDiagramAndStopListPage
-              lineDiagram={lineState.data}
-              route={route}
-              directionId={state.directionId}
-            />
-          )
-        : null}
+
+      {/* OLD LINE DIAGRAM */}
```

Comment:
My plan—either in this PR or a follow-up—is to remove all of the old components and rename the new ones to remove the "Tree" prefix. I thought that at least for the initial review though it might be helpful to leave them for reference since most of this is taken as directly as possible.